spec/support/features/checkouts_helper.rb

```diff
@@ -0,0 +1,13 @@
+module Features
+  def have_checked_out_book(title)
+    have_css('ul.books.unavailable li', text: title)
+  end
+
+  def have_returned_book(title)
+    have_css('ul.books.available li', text: title)
+  end
+
+  def have_available_book(title)
```

Comment:
I think that's ok. I'd perhaps consider using [`alias_method`](http://ruby-doc.org/core-2.4.1/Module.html#method-i-alias_method) rather than duplicating the definition. RSpec itself has many aliases for its matchers. On one hand it can be confusing, on the other hand sometimes being able to choose between them can really aid in readability/understandability